lib/ae_mdw/db/mutations/contract_call_mutation.ex

```diff
@@ -30,13 +43,47 @@ defmodule AeMdw.Db.ContractCallMutation do
 
   @spec mutate(t()) :: :ok
   def mutate(%__MODULE__{
+        contract_pk: contract_pk,
+        caller_pk: caller_pk,
         create_txi: create_txi,
         txi: txi,
         fun_arg_res: fun_arg_res,
         call_rec: call_rec
       }) do
     DBContract.call_write(create_txi, txi, fun_arg_res)
     DBContract.logs_write(create_txi, txi, call_rec)
+
+    with true <- Contract.is_aex9?(contract_pk),
+         {:ok, method_name, method_args} <-
+           Contract.extract_non_stateful_aex9_function(fun_arg_res) do
+      update_aex9_presence(
+        contract_pk,
+        caller_pk,
+        txi,
+        method_name,
+        method_args
+      )
+    end
+
+    :ok
+  end
+
+  #
+  # Private functions
+  #
+  defp update_aex9_presence(contract_pk, caller_pk, txi, method_name, method_args) do
+    account_pk =
+      if method_name in ["burn", "swap"] do
+        caller_pk
```

Comment:
only swap receiver would be interesting, but not part of the standard

Reply:
Could you explain why the `burn` should be ignored? Asking because they are in the same section of the standard.

Reply:
both don't need to be tracked

Reply:
For the current model it would be needed for the same reason as:
https://github.com/aeternity/ae_mdw/pull/438#discussion_r783353246

Unless your request aims to omit this particular changes (after a burn or swap) from /aex9/balances/account/. 

Could you detail it please?